### PR TITLE
Add tab spinner severity job

### PR DESCRIPTION
**Pull request by fbertsch**
Tested this, ran successfully in ~5.5 hours on a 12 node cluster.

### CODE REVIEW

File: dags/spinner_severity_generator.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
+from airflow import DAG
+from datetime import datetime, timedelta
+from operators.emr_spark_operator import EMRSparkOperator
+
+default_args = {
+    'owner': 'frank@mozilla.com',
+    'depends_on_past': False,
+    'start_date': datetime(2017, 4, 25),
```
Comment (vitillo) [inline dags/spinner_severity_generator.py]: Is the backfill needed?

**Comment by fbertsch** [comment]
@vitillo it does not need backfill, updated.

File: jobs/run-severity-generator.sh [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+pip install .
+python setup.py bdist_egg
+
+rm -rf output
+mkdir output
+
+spark-submit scheduling/tab_spinner.py
+
+# upload files to s3
+aws s3 sync ./output s3://telemetry-public-analysis-2/spinner-severity-generator/data/
```
Comment (vitillo) [inline jobs/run-severity-generator.sh]: Why are files manually uploaded instead of doing this in the job itself?

Comment (fbertsch) [inline jobs/run-severity-generator.sh]: I don't have a good reason for this. It just happened this way as I moved the ipynb over.

Comment (vitillo) [inline jobs/run-severity-generator.sh]: OK, it would be great if you could move that logic into the job itself.

**Comment by fbertsch** [comment]
@vitillo moved file copying to the job. RFAL

**Pull request merged**